content/api/commands/url.md

```diff
@@ -104,11 +104,11 @@ return them the full current URL. We almost never refer to the URL as an `href`.
 
 #### Hardcoded versus using the configuration object
 
-Instead of hardcoding the URL you can use the `baseUrl` of the
-[Cypress configuration](/guides/references/configuration).
+Instead of hard-coding the URL used in the assertion, you can use the `baseUrl` the
+defined in the [Cypress configuration](/guides/references/configuration).
```

Comment:
Is one method preferred over the other? Why?

Reply:
Added a snippet to point to the best practices guide / you-tube video that describe why you should set the baseUrl: [`f3ebeb4` (#4202)](https://github.com/cypress-io/cypress-documentation/pull/4202/commits/f3ebeb460b51d21f6be0b2245b08c68a4a9461e4#diff-6680016ffcad5d4eb739ec1fba3e872c92227b9a4ecea95e2cb018793fa202cdR107)